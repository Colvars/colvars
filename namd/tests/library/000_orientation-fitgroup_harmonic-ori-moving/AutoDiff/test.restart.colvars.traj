# step         one                                                                                               fa_one                                                                                             E_harmonic1           x0_one                                                                                            W_harmonic1          
          20   (  9.99760693818246e-01 ,  1.39216454886428e-02 ,  1.19362887828950e-02 , -1.19276105335207e-02 ) ( -7.44154910910036e-05 , -1.03623406304924e-06 ,  6.23499480534843e-03 ,  8.87811454163381e-07 )   4.86008746875330e-03 (  0.00000000000000e+00 ,  0.00000000000000e+00 ,  1.00000000000000e+00 ,  0.00000000000000e+00 )  2.27813232055700e-03
          21   (  9.99758313478268e-01 ,  1.37518526317376e-02 ,  1.20943745877787e-02 , -1.21625360684149e-02 ) ( -7.53933764660893e-05 , -1.03704924339528e-06 ,  6.23435061795005e-03 ,  9.17196334580247e-07 )   4.85910171318368e-03 (  0.00000000000000e+00 ,  0.00000000000000e+00 ,  1.00000000000000e+00 ,  0.00000000000000e+00 )  2.27813232055700e-03
          22   (  9.99753796094871e-01 ,  1.36537839555646e-02 ,  1.22845326108695e-02 , -1.24503669071325e-02 ) ( -7.65692635273181e-05 , -1.04571764160572e-06 ,  6.23357553386989e-03 ,  9.53550192505157e-07 )   4.85791609892022e-03 (  0.00000000000000e+00 ,  0.00000000000000e+00 ,  1.00000000000000e+00 ,  0.00000000000000e+00 )  2.27813232055700e-03
          23   (  9.99747426449245e-01 ,  1.36276760151045e-02 ,  1.24954695687563e-02 , -1.27762668567522e-02 ) ( -7.78731950371622e-05 , -1.06149877874316e-06 ,  6.23271549315738e-03 ,  9.95180076948371e-07 )   4.85660109628904e-03 (  0.00000000000000e+00 ,  0.00000000000000e+00 ,  1.00000000000000e+00 ,  0.00000000000000e+00 )  2.27813232055700e-03
          24   (  9.99739603092028e-01 ,  1.36692768838184e-02 ,  1.27143928598848e-02 , -1.31233034359725e-02 ) ( -7.92260158564982e-05 , -1.08324442064198e-06 ,  6.23182259923561e-03 ,  1.03997785312531e-06 )   4.85523649061308e-03 (  0.00000000000000e+00 ,  0.00000000000000e+00 ,  1.00000000000000e+00 ,  0.00000000000000e+00 )  2.27813232055700e-03
          25   (  9.99730830850349e-01 ,  1.37696650158448e-02 ,  1.29278262893479e-02 , -1.34734360893895e-02 ) ( -8.05444416230567e-05 , -1.10936858783720e-06 ,  6.23095181081672e-03 ,  1.08550257036763e-06 )   4.85390628560139e-03 (  0.00000000000000e+00 ,  0.00000000000000e+00 ,  1.00000000000000e+00 ,  0.00000000000000e+00 )  2.27813232055700e-03
          26   (  9.99721696829550e-01 ,  1.39157668005032e-02 ,  1.31225150492047e-02 , -1.38086900675579e-02 ) ( -8.17466563544410e-05 , -1.13788408329726e-06 ,  6.23015725331999e-03 ,  1.12912848169394e-06 )   4.85269306082901e-03 (  0.00000000000000e+00 ,  0.00000000000000e+00 ,  1.00000000000000e+00 ,  0.00000000000000e+00 )  2.27813232055700e-03
          27   (  9.99712830408440e-01 ,  1.40915219271310e-02 ,  1.32863099760682e-02 , -1.41123950392548e-02 ) ( -8.27577572531398e-05 , -1.16651773939557e-06 ,  6.22948859730489e-03 ,  1.16824564754451e-06 )   4.85167246948734e-03 (  0.00000000000000e+00 ,  0.00000000000000e+00 ,  1.00000000000000e+00 ,  0.00000000000000e+00 )  2.27813232055700e-03
          28   (  9.99704851538043e-01 ,  1.42794923087412e-02 ,  1.34089250569295e-02 , -1.43703388798510e-02 ) ( -8.35143963095269e-05 , -1.19289526097260e-06 ,  6.22898794027442e-03 ,  1.20048449746711e-06 )   4.85090853482966e-03 (  0.00000000000000e+00 ,  0.00000000000000e+00 ,  1.00000000000000e+00 ,  0.00000000000000e+00 )  2.27813232055700e-03
          29   (  9.99698314859356e-01 ,  1.44626159417893e-02 ,  1.34824987759774e-02 , -1.45716929091923e-02 ) ( -8.39681999132023e-05 , -1.21476630361119e-06 ,  6.22868748240004e-03 ,  1.22392806418303e-06 )   4.85045017320758e-03 (  0.00000000000000e+00 ,  0.00000000000000e+00 ,  1.00000000000000e+00 ,  0.00000000000000e+00 )  2.27813232055700e-03
          30   (  9.99693658824898e-01 ,  1.46257926205038e-02 ,  1.35019394575005e-02 , -1.47095998041003e-02 ) ( -8.40878559230599e-05 , -1.23022841225092e-06 ,  6.22860808561504e-03 ,  1.23727773812928e-06 )   4.85032906200261e-03 (  0.00000000000000e+00 ,  0.00000000000000e+00 ,  1.00000000000000e+00 ,  0.00000000000000e+00 )  2.27813232055700e-03
          31   (  9.99691168442251e-01 ,  1.47570480953222e-02 ,  1.34650761492187e-02 , -1.47813732236213e-02 ) ( -8.38600119325431e-05 , -1.23790853458392e-06 ,  6.22875863535765e-03 ,  1.23994907031517e-06 )   4.85055871362938e-03 (  0.00000000000000e+00 ,  0.00000000000000e+00 ,  1.00000000000000e+00 ,  0.00000000000000e+00 )  2.27813232055700e-03
          32   (  9.99690957546167e-01 ,  1.48481276856586e-02 ,  1.33726578643432e-02 , -1.47883239855450e-02 ) ( -8.32892561630635e-05 , -1.23707181806292e-06 ,  6.22913603470891e-03 ,  1.23208927254656e-06 )   4.85113448593648e-03 (  0.00000000000000e+00 ,  0.00000000000000e+00 ,  1.00000000000000e+00 ,  0.00000000000000e+00 )  2.27813232055700e-03
          33   (  9.99692972301071e-01 ,  1.48944940900112e-02 ,  1.32282447187099e-02 , -1.47352882331596e-02 ) ( -8.23974522868586e-05 , -1.22764528722624e-06 ,  6.22972565464919e-03 ,  1.21452310135798e-06 )   4.85203425681462e-03 (  0.00000000000000e+00 ,  0.00000000000000e+00 ,  1.00000000000000e+00 ,  0.00000000000000e+00 )  2.27813232055700e-03
          34   (  9.99697013007692e-01 ,  1.48948311570748e-02 ,  1.30380222677718e-02 , -1.46299748341174e-02 ) ( -8.12226196305917e-05 , -1.21016386944397e-06 ,  6.23050211131118e-03 ,  1.18864502513774e-06 )   4.85321956879579e-03 (  0.00000000000000e+00 ,  0.00000000000000e+00 ,  1.00000000000000e+00 ,  0.00000000000000e+00 )  2.27813232055700e-03
          35   (  9.99702767610873e-01 ,  1.48502632434009e-02 ,  1.28105508103196e-02 , -1.44822615878513e-02 ) ( -7.98174260893882e-05 , -1.18566220604831e-06 ,  6.23143031865448e-03 ,  1.15628052791932e-06 )   4.85463717256666e-03 (  0.00000000000000e+00 ,  0.00000000000000e+00 ,  1.00000000000000e+00 ,  0.00000000000000e+00 )  2.27813232055700e-03
          36   (  9.99709848575880e-01 ,  1.47635678298615e-02 ,  1.25564420596702e-02 , -1.43035480957163e-02 ) ( -7.82472390297188e-05 , -1.15554370356586e-06 ,  6.23246684237558e-03 ,  1.11953798235853e-06 )   4.85622102004666e-03 (  0.00000000000000e+00 ,  0.00000000000000e+00 ,  1.00000000000000e+00 ,  0.00000000000000e+00 )  2.27813232055700e-03
          37   (  9.99717825758149e-01 ,  1.46386420259571e-02 ,  1.22879416152389e-02 , -1.41062224530848e-02 ) ( -7.65875917949357e-05 , -1.12145478557008e-06 ,  6.23356163672723e-03 ,  1.08066654326796e-06 )   4.85789484544943e-03 (  0.00000000000000e+00 ,  0.00000000000000e+00 ,  1.00000000000000e+00 ,  0.00000000000000e+00 )  2.27813232055700e-03
          38   (  9.99726250964308e-01 ,  1.44803785129685e-02 ,  1.20183947274420e-02 , -1.39032355639129e-02 ) ( -7.49209173989000e-05 , -1.08518030953821e-06 ,  6.23466024929299e-03 ,  1.04192839015343e-06 )   4.85957547890113e-03 (  0.00000000000000e+00 ,  0.00000000000000e+00 ,  1.00000000000000e+00 ,  0.00000000000000e+00 )  2.27813232055700e-03
          39   (  9.99734673870441e-01 ,  1.42949388301149e-02 ,  1.17615869623806e-02 , -1.37077225363170e-02 ) ( -7.33324894756701e-05 , -1.04856166212217e-06 ,  6.23570652181050e-03 ,  1.00548820092254e-06 )   4.86117694883134e-03 (  0.00000000000000e+00 ,  0.00000000000000e+00 ,  1.00000000000000e+00 ,  0.00000000000000e+00 )  2.27813232055700e-03
          40   (  9.99742652209286e-01 ,  1.40902435879328e-02 ,  1.15309792246189e-02 , -1.35325868663831e-02 ) ( -7.19056857990176e-05 , -1.01342943209091e-06 ,  6.23664570367131e-03 ,  9.73320421122090e-07 )   4.86261525439548e-03 (  0.00000000000000e+00 ,  0.00000000000000e+00 ,  1.00000000000000e+00 ,  0.00000000000000e+00 )  2.27813232055700e-03
