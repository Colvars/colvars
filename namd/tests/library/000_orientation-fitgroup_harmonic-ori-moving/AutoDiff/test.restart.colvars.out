colvars: ----------------------------------------------------------------------
colvars: Please cite Fiorin et al, Mol Phys 2013:
colvars:  https://dx.doi.org/10.1080/00268976.2013.813594
colvars: in any publication based on this calculation.
colvars: SMP parallelism is enabled; if needed, use "smp off" to override this.
colvars: This version was built with the C++11 standard or higher.
colvars: ----------------------------------------------------------------------
colvars: Reading new configuration from file "test.in":
colvars: # units = "" [default]
colvars: # indexFile = "index.ndx"
colvars: The following index groups are currently defined:
colvars:   Protein (104 atoms)
colvars:   Protein_noH (51 atoms)
colvars:   Protein_Backbone (40 atoms)
colvars:   Protein_C-alpha (10 atoms)
colvars:   RMSD_atoms (10 atoms)
colvars:   Protein_C-alpha_1_2 (2 atoms)
colvars:   Protein_C-alpha_9_10 (2 atoms)
colvars:   Protein_C-alpha_1 (1 atoms)
colvars:   group1 (4 atoms)
colvars:   Protein_C-alpha_2 (1 atoms)
colvars:   group2 (4 atoms)
colvars:   Protein_C-alpha_3 (1 atoms)
colvars:   group3 (4 atoms)
colvars:   Protein_C-alpha_4 (1 atoms)
colvars:   group4 (4 atoms)
colvars:   Protein_C-alpha_5 (1 atoms)
colvars:   group5 (4 atoms)
colvars:   Protein_C-alpha_6 (1 atoms)
colvars:   group6 (4 atoms)
colvars:   Protein_C-alpha_7 (1 atoms)
colvars:   group7 (4 atoms)
colvars:   Protein_C-alpha_8 (1 atoms)
colvars:   group8 (4 atoms)
colvars:   Protein_C-alpha_9 (1 atoms)
colvars:   group9 (4 atoms)
colvars:   Protein_C-alpha_10 (1 atoms)
colvars:   group10 (4 atoms)
colvars:   heavy_atoms (51 atoms)
colvars: # smp = on [default]
colvars: # colvarsTrajFrequency = 1
colvars: # colvarsRestartFrequency = 10
colvars: # scriptedColvarForces = off [default]
colvars: # scriptingAfterBiases = off [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "one"
colvars:   Initializing a new "orientation" component.
colvars:     # name = "" [default]
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:       Initializing atom group "atoms".
colvars:       # name = "" [default]
colvars:       # centerReference = on
colvars:       # rotateReference = on
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "RMSD_atoms"
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       Within atom group "atoms":
colvars:       Initializing atom group "fittingGroup".
colvars:       # name = "" [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "heavy_atoms"
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "fittingGroup" defined with 51 atoms requested: total mass = 674.397, total charge = -7.64.
colvars:       # refPositions =  [default]
colvars:       # refPositionsFile = "heavy_atoms_refpos.xyz"
colvars:       # refPositionsCol = "" [default]
colvars:       Warning: beginning from 2019-11-26 the XYZ file reader assumes Angstrom units.
colvars:       Warning: atom group "atoms" will be aligned to a fixed orientation given by the reference positions provided.  If the internal structure of the group changes too much (i.e. its RMSD is comparable to its radius of gyration), the optimal rotation and its gradients may become discontinuous.  If that happens, use fittingGroup (or a different definition for it if already defined) to align the coordinates.
colvars:       # enableFitGradients = off
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "atoms" defined with 10 atoms requested: total mass = 120.11, total charge = 0.53.
colvars:     # refPositions =  [default]
colvars:     # refPositionsFile = "rmsd_atoms_refpos.xyz"
colvars:     # refPositionsCol = "" [default]
colvars:     Centering the reference coordinates: it is assumed that each atom is the closest periodic image to the center of geometry.
colvars:     # closestToQuaternion = ( 1 , 0 , 0 , 0 ) [default]
colvars:   All components initialized.
colvars:   # timeStepFactor = 1 [default]
colvars:   # width = 0.5
colvars:   # hardLowerBoundary = off [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputTotalForce = off [default]
colvars:   # outputAppliedForce = on
colvars:   # subtractAppliedForce = off [default]
colvars:   # runAve = off [default]
colvars:   # corrFunc = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "harmonic" instance.
colvars:   # name = "harmonic1" [default]
colvars:   # colvars = { one }
colvars:   # stepZeroData = off [default]
colvars:   # outputEnergy = on
colvars:   # outputFreq = 10 [default]
colvars:   # timeStepFactor = 1 [default]
colvars:   # writeTISamples = off [default]
colvars:   # writeTIPMF = off [default]
colvars:   # centers = { ( 1 , 0 , 0 , 0 ) }
colvars:   # targetCenters = { ( 0 , 0 , 1 , 0 ) }
colvars:   # targetNumSteps = 10
colvars:   # targetNumStages = 0 [default]
colvars:   # outputAccumulatedWork = on
colvars:   # outputCenters = on
colvars:   # forceConstant = 0.001
colvars:   # targetForceConstant = -1 [default]
colvars:   The force constant for colvar "one" will be rescaled to 0.004 according to the specified width (0.5).
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module (re)initialized.
colvars: ----------------------------------------------------------------------
colvars: Updating NAMD interface:
colvars: updating atomic data (51 atoms).
colvars: updating group data (0 scalable groups, 0 atoms in total).
colvars: Re-initialized atom group for variable "one":0/0. 10 atoms: total mass = 120.11, total charge = 0.53.
colvars: ----------------------------------------------------------------------
colvars: Restarting from file "test.colvars.state".
colvars:   Restarting collective variable "one" from value: ( 0.999761 , 0.0139216 , 0.0119363 , -0.0119276 )
colvars:   Restarting harmonic bias "harmonic1" from step number 20.
colvars: ----------------------------------------------------------------------
colvars: The restart output state file will be "test.restart.tmp.colvars.state".
colvars: The final output state file will be "test.restart.colvars.state".
colvars: Opening trajectory file "test.restart.colvars.traj".
colvars: Redefining the Tcl "cv" command to the new script interface.
colvars: Updating NAMD interface:
colvars: updating atomic data (51 atoms).
colvars: updating group data (0 scalable groups, 0 atoms in total).
colvars: Re-initialized atom group for variable "one":0/0. 10 atoms: total mass = 120.11, total charge = 0.53.
colvars: The restart output state file will be "test.restart.tmp.colvars.state".
colvars: The final output state file will be "test.restart.colvars.state".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.restart.colvars.traj".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.restart.colvars.traj".
colvars: Saving collective variables state to "test.restart.tmp.colvars.state".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.restart.colvars.traj".
colvars: Saving collective variables state to "test.restart.tmp.colvars.state".
colvars: Saving collective variables state to "test.restart.colvars.state".
